community/values/src/test/java/org/neo4j/values/ValueWriteToTest.java

```diff
@@ -70,36 +71,15 @@
                 shouldWrite( new String[]{"a", "b"}, beginArray( 2, STRING ), "a", "b", endArray() ),
                 shouldWrite( new boolean[]{true, false}, beginArray( 2, BOOLEAN ), true, false, endArray() ),
 
-                // lazy array properties
-                shouldWrite( Values.lazyByteArray( () -> new byte[]{1, 2, 3} ),
-                        byteArray( new byte[]{1, 2, 3} ) ),
-
-                shouldWrite( Values.lazyShortArray( () -> new short[]{1, 2, 3} ),
-                        beginArray( 3, SHORT ), (short) 1, (short) 2, (short) 3, endArray() ),
-
-                shouldWrite( Values.lazyIntArray( () -> new int[]{1, 2, 3} ),
-                        beginArray( 3, INT ), 1, 2, 3, endArray() ),
-
-                shouldWrite( Values.lazyLongArray( () -> new long[]{1, 2, 3} ),
-                        beginArray( 3, LONG ), 1L, 2L, 3L, endArray() ),
-
-                shouldWrite( Values.lazyFloatArray( () -> new float[]{1, 2, 3} ),
-                        beginArray( 3, FLOAT ), 1.0f, 2.0f, 3.0f, endArray() ),
-
-                shouldWrite( Values.lazyDoubleArray( () -> new double[]{1, 2, 3} ),
-                        beginArray( 3, DOUBLE ), 1.0, 2.0, 3.0, endArray() ),
-
-                shouldWrite( Values.lazyCharArray( () -> new char[]{'a', 'b'} ),
-                        beginArray( 2, CHAR ), 'a', 'b', endArray() ),
-
-                shouldWrite( Values.lazyStringArray( () -> new String[]{"a", "b"} ),
-                        beginArray( 2, STRING ), "a", "b", endArray() ),
-
-                shouldWrite( Values.lazyBooleanArray( () -> new boolean[]{true, false} ),
-                        beginArray( 2, BOOLEAN ), true, false, endArray() ),
-
-                // lazy string property
-                shouldWrite( Values.lazyStringValue( () -> "Finnlines" ), "Finnlines" )
+                shouldWrite( new byte[]{1,2,3}, beginArray( 3, BYTE ), (byte)1, (byte)2, (byte)3, endArray() ),
+                shouldWrite( new short[]{1,2,3}, beginArray( 3, SHORT ), (short)1, (short)2, (short)3, endArray() ),
+                shouldWrite( new int[]{1,2,3}, beginArray( 3, INT ), 1, 2, 3, endArray() ),
+                shouldWrite( new long[]{1,2,3}, beginArray( 3, LONG ), 1L, 2L, 3L, endArray() ),
+                shouldWrite( new float[]{1,2,3}, beginArray( 3, FLOAT ), 1.0f, 2.0f, 3.0f, endArray() ),
+                shouldWrite( new double[]{1,2,3}, beginArray( 3, DOUBLE ), 1.0, 2.0, 3.0, endArray() ),
+                shouldWrite( new char[]{'a','b'}, beginArray( 2, CHAR ), 'a', 'b', endArray() ),
+                shouldWrite( new String[]{"a","b"}, beginArray( 2, STRING ), "a", "b", endArray() ),
+                shouldWrite( new boolean[]{true, false}, beginArray( 2, BOOLEAN), true, false, endArray() )
```

Comment:
These tests already exists above ^^. Also the byte array is special (writes `byte[]` directly), so I hope this test should fail? Otherwise something is broken with the asserts.